docs/framework/security/wif-claims-programming-model.md

```diff
@@ -17,7 +17,7 @@ ASP.NET and Windows Communication Foundation (WCF) developers ordinarily use the
 
 - <xref:System.Security.Claims.Claim.Value%2A> contains the value of the claim and is represented as a string. For example, the email address can be represented as "someone@contoso.com".
 
-- <xref:System.Security.Claims.Claim.ValueType%2A> represents the type of the claim value and is typically a URI. For example, the string type is represented as `http://www.w3.org/2001/XMLSchema#string`. The value type must be a QName according to the XML schema. The value should be of the format `namespace#format` to enable WIF to output a valid QName value. If the namespace is not a well-defined namespace, the generated XML probably cannot be schema validated, because there will not be a published XSD file for that namespace. The default value type is `http://www.w3.org/2001/XMLSchema#string`. Please see [http://www.w3.org/2001/XMLSchema](https://go.microsoft.com/fwlink/?LinkId=209155) for well-known value types that you can use safely.
+- <xref:System.Security.Claims.Claim.ValueType%2A> represents the type of the claim value and is typically a URI. For example, the string type is represented as `https://www.w3.org/2001/XMLSchema#string`. The value type must be a QName according to the XML schema. The value should be of the format `namespace#format` to enable WIF to output a valid QName value. If the namespace is not a well-defined namespace, the generated XML probably cannot be schema validated, because there will not be a published XSD file for that namespace. The default value type is `https://www.w3.org/2001/XMLSchema#string`. Please see [https://www.w3.org/2001/XMLSchema](https://go.microsoft.com/fwlink/?LinkId=209155) for well-known value types that you can use safely.
```

Comment:
I'd keep the values not in active links (inside backticks) still using http. 

Please change the last sentence to:
`For information about well-known value types that you can use safely, see the [W3C XML Schema](https://www.w3.org/2001/XMLSchema) page.`